### PR TITLE
Add message about beta

### PR DESCRIPTION
**Pull request by MichalCz**
This adds a small log file to let users know that they can
now enrol to the waiting list for our Beta platform.﻿

### CODE REVIEW

**Review by bizconnectme — CHANGES_REQUESTED** [review]
Add better marketing infos

**Review by bizconnectme — CHANGES_REQUESTED** [review]
New tips

**Review by alicja-gruzdz — APPROVED** [review]
It's a subtle banner.

It's a matter of personal taste but I prefer the message to be printed together with extra.
Also, without diving the sentence.


Verified :heavy_check_mark: 
Zrzut ekranu z 2022-04-26 17-16-46

File: packages/sth/src/bin/hub.ts [hunk 1]
```diff
@@ -87,10 +87,20 @@ configService.update({
     }
 });
 
+const tips = [
+    ["Run sequences in our cloud.", { "Check it": "out as a beta tester", here: "https://scr.je/join-beta-sth" }],
+    ["Now you can run sequences in the cloud and deploy them to multiple locations simultaneously", { "check the": "beta version", "at-this-link": "https://scr.je/join-beta-sth" }],
+    ["You don't need to maintain your own server anymore", { "Check out": "Scramjet Cloud Platform", here: "https://scr.je/join-beta-sth" }]
+];
```
Comment (a-tylenda) [inline packages/sth/src/bin/hub.ts:94]:
```suggestion
const tips = [
    ["Run Sequences in our cloud.", { "Check it": "out as a beta tester", here: "https://scr.je/join-beta-sth" }],
    ["Now you can run Sequences in the cloud and deploy them to multiple locations simultaneously", { "check the": "beta version", "at-this-link": "https://scr.je/join-beta-sth" }],
    ["You don't need to maintain your own server anymore", { "Check out": "Scramjet Cloud Platform", here: "https://scr.je/join-beta-sth" }]
];
```

**Review by a-tylenda — APPROVED** [review]

**Pull request merged**